terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/time/JdbcClockFactory.java

```diff
@@ -69,8 +68,11 @@ public Clock tick(ZoneId zone) {
      * @return instant of date and time
      */
     private Instant instant(ZoneId zone) {
-        return jdbcTemplate.queryForObject(currentTimestampQuery, //
-                (rs, rowNum) -> rs.getTimestamp(1).toLocalDateTime().atZone(zone)) //
-                .toInstant();
+        Timestamp timestamp = jdbcTemplate.queryForObject(currentTimestampQuery,
```

Comment:
- [ ] This is not a bug report, but it is flagged as a code smell with the following suggestion:  
> Declare this local variable with 'var' instead.

Reply:
Since the return type of `queryForObject()` is not self-explanatory, we will not use var.